WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3314,6 +3400,41 @@ public void onPostChanged(OnPostChanged event) {
             if (!event.isError()) {
                 // here update the menu if it's not a draft anymore
                 invalidateOptionsMenu();
+
+                if (mIsUpdatingPost) {
+                    mIsUpdatingPost = false;
+                    mPost = mPostStore.getPostByLocalPostId(mPost.getId());
+                    refreshEditorContent();
+
+                    if (mViewPager != null) {
+                        Snackbar.make(mViewPager, getString(R.string.local_changes_discarded), Snackbar.LENGTH_LONG)
```

Comment:
Ahh, I always forget about `AccessibilityUtils`.  Thanks for the reminder.  I'll update that in a subsequent pull request since this one has been merged.